cbtdoc.go

```diff
@@ -90,40 +93,48 @@ Example:
 
 cbt -project my-project -instance my-instance lookup my-table $'\224\257\312W\365:\205d\333\2471\315\'
 
+
 For convenience, you can add values for the -project, -instance, -creds, -admin-endpoint and -data-endpoint
 options to your ~/.cbtrc file in the following format:
 
-	project = my-project-123
-	instance = my-instance
-	creds = path-to-account-key.json
-	admin-endpoint = hostname:port
-	data-endpoint = hostname:port
-	auth-token = AJAvW039NO1nDcijk_J6_rFXG_...
-	timeout = 30s
+
+    project = my-project-123
+    instance = my-instance
+    creds = path-to-account-key.json
+    admin-endpoint = hostname:port
+    data-endpoint = hostname:port
+    auth-token = AJAvW039NO1nDcijk_J6_rFXG_...
+    timeout = 30s
 
 All values are optional and can be overridden at the command prompt.
 
-# Count rows in a table
 
-Usage:
 
-	cbt count <table-id> [prefix=<row-key-prefix>]
 
-# Create app profile for an instance
+Count rows in a table
```

Comment:
Why did we remove the `#`?  is it intentional?

Reply:
:shrug: thats just what I got when I ran `go generate`.  I'm not sure the logic of any of it :P